packages/tsdocs/README.md

```diff
@@ -0,0 +1,58 @@
+# @loopback/tsdocs
+
+This module provides [API docs](https://github.com/Microsoft/tsdoc) generation
+for `@loopback/*` packages.
+
+It's built on top of https://api-extractor.com/:
+
+- https://github.com/Microsoft/web-build-tools/tree/master/apps/api-extractor
+- https://github.com/Microsoft/web-build-tools/tree/master/apps/api-documenter
+
+## Basic Use
+
+### Build api reports and doc models
+
+```sh
+npm start
+```
+
+The command above will traverse all TypeScript packages in the monorepo and run
+`api-extractor` to generate api reports and doc models into
+`loopback-next/docs/apidocs`:
+
+- `reports`: api reports
+- `reports-temp`: temporary api reports
+- `models`: doc models
+
+### Generate api docs as markdown files
+
+```sh
+npm run generate-apidocs
```

Comment:
While this may work inside loopback-next monorepo, I don't think it will work for people installing `@loopback/tsdocs` into their own projects. I see that the packages is marked as `private: true`, which means it's not intended for usage outside of loopback-next monorepo. Can you please mention this in package's README too?

If this package is private, how do you envision to generate API docs for experimental packages in loopback-labs?  Wouldn't it better if `@loopback/tsdocs` was a generic tool that can be used in any monorepo? Maybe not, I am just asking.